### DISCUSSION TITLE
Support multiple datasources for same alert rule

### DISCUSSION THREAD

**Discussion by kthommandra** [grafana/grafana#66125] [Alerting]
Dashboards support datasource as a variable so that the same panel can be targeted to different clusters.

Alert rule creation does not seem to offer using a set of datasources. Thus an alert needs to be duplicated for each cluster.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82758. Feel free to continue the discussion around this there. Thank you!
